go-controller/pkg/node/gateway_localnet.go

```diff
@@ -3,7 +3,9 @@
 package node
 
 import (
+	//"debug/elf"
```

Comment:
leftover from debugging?

Reply:
It was there before and hence I did not remove it. I suppose I can remove it.